server.py

```diff
@@ -21,6 +21,15 @@
 import threading
 import traceback
 
+## Utilities
+
+def unique(lst):
+    new_lst = []
+    for item in lst :
+        if item not in new_lst :
```

Comment:
Please don't put spaces before colons. This violates PEP 8.

I suggest configuring your editor to run [flake8](http://flake8.pycqa.org/en/latest/), [pylint](https://www.pylint.org/), or a similar tool to avoid this in the future.